examples/webui-ext/readme.md

```diff
@@ -0,0 +1,53 @@
+# Web UI Extensions in Sidekiq 7.3 and 8.0
+
+This is an example of how to extend Sidekiq's Web UI to add your own tab.
+
+## Initialization
+
+The Ruby code for your extension goes in `lib/` like any normal Rubygem.
+When the user includes the `my-gem` gem within their bundle, Bundler will call `require "my-gem"` which corresponds to `lib/my-gem.rb` in your project.
+This file should **not** require any web code or assets because you don't know if you're running within a Sidekiq process or a Web process.
+
+> Don't require your web extensions in Sidekiq's initializer. This is a common mistake.
+
+Your web extension should be activated by the user calling `require "my-gem/web"` in Rails' `config/routes.rb` as that guarantees we are in a Web process.
+
+## Registration
+
+Your Web extension must register itself with Sidekiq::Web to appear in the Sidekiq UI.
+See `lib/sidekiq-redis_info/web.rb` for a full-featured example.
+
+## Content Security
+
+Sidekiq 7.3 and 8.0 explicitly lock down the Web UI to prevent XSS and malicious content injection.
+In Sidekiq 7.3, the Web UI no longer allows ěmbedding JavaScript directly on the HTML page via
+`<script>...code...</script>`. In Sidekiq 8.0, inline styling via `<style>...css...</style>` will not work.
```

Comment:
A more common example (I think) would be something like `<div style="background=red">Hello World!</div>`. Listing both seems fine. You may also call out dynamic js like `myDiv.style.background = "red"`